problems/5.loops.js

```diff
@@ -23,7 +34,12 @@ function multiplesToOneThousand() {
 * @returns {number[]} - An array with the first {count} powers of two
 */
 function powersOfTwo() {
-
+const powersOfTwo = count => {
```

Comment:
Lots of issues on the loops. Please go back to the loops lab and work on them. Thanks!